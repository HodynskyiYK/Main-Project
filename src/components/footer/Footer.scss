.footer {

  .footer-content {
    background: $grey;
  }

  .copyright {
    color: $red_color;
    font: 300 1.25rem/1.2em $main_font;
    text-align: center;
    margin: 0;
    padding: 1.5rem 0;

    b {
      font-weight: 900
    }
  }
}