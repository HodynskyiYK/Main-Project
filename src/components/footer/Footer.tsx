import React, {FC} from 'react'

export const Footer: FC = () => {

    return (
        <footer className="footer">
            <div className="container footer-content">
                <p className="copyright">
                    <b>netflix</b>roulette
                </p>
            </div>
        </footer>
    )
}
