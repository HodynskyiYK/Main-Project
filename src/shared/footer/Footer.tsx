import React, {FC} from 'react'
import classnames from 'classnames'
import styles from './Footer.module.scss'
import { Logo } from '../../components/logo'

export const Footer: FC = () => {

    return (
        <footer className={styles.footer}>
            <div className={classnames('container', styles.footerContent)}>
                <p className={styles.copyright}>
                    <Logo/>
                </p>
            </div>
        </footer>
    )
}
