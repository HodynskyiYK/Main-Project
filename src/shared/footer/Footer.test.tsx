import React from 'react'
import { render } from '@testing-library/react'
import { Footer } from './Footer'

describe('Footer tests', () => {

    test('To match snapshot', () => {
        const {asFragment} = render(<Footer/>)
        expect(asFragment()).toMatchSnapshot()
    })
})