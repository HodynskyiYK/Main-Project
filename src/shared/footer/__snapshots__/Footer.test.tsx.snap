// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer tests To match snapshot 1`] = `
<DocumentFragment>
  <footer
    class="footer"
  >
    <div
      class="container footerContent"
    >
      <p
        class="copyright"
      >
        <span
          class="Logo"
        >
          <b>
            netflix
          </b>
          roulette
        </span>
      </p>
    </div>
  </footer>
</DocumentFragment>
`;
